packages/algorand-js/src/runtime/runtime.ts

```diff
@@ -194,7 +211,7 @@ export class Runtime {
             break;
           }
           case txnParam.toAccountAddr: {
-            account.amount += txnParam.amountMicroAlgos; // add 'x' algo to receiver
+            account.amount += txnParam.amountMicroAlgos as number; // add 'x' algo to receiver
```

Comment:
We should use big numbers here, because we need to support uint64.

Reply:
Sorry, i don't follow. This amount will be passed by the user via `ExecParams` and not TEAL code. So it's maximum value should be javascript max number. (`Amount must be a positive number and smaller than 2^53-1`)

Reply:
So the argument type should be `number | BigInt`
```
amountMicroAlgos: number | BigInt
```

Reply:
Alright, i'll update the `ExecParams` type. But i still don't see user passing `BigInt(..)` as transfer amount :/